### DISCUSSION TITLE
Map one key to multiple commands

### DISCUSSION THREAD

**Discussion by fritzrehde** [qutebrowser/qutebrowser#7042] [Q&A] [answered]
I wanted to ask if it is possible to map one key to multiple commands. I would like to have a command that reloads the config file and, at the same time, shows a message that it is reloaded. Is this possible?
Thank you very much!

**Comment by toofar** [comment, marked as answer]
You can use `;;` separate commands. For example `message-info "reloading config.py" ;; config-source`.

**Reply by fritzrehde** [reply]
Thank you, it works! Is there a way to change the length (in seconds) of messages?

Edit: I just found messages.timeout in the docs. This sets the length for all messages though, right? Is there a way to set the length for a specific use of a method (e.g. can I set the length only for the config reloaded message)?

**Reply by toofar** [reply]
Nope, there is no way to do per-message timing
